src/herder/test/HerderTests.cpp

```diff
@@ -1412,9 +1406,7 @@ testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps,
             // Confirm that trimInvalid() as used by
             // HerderImpl::triggerNextLedger() trims the transaction if and only
             // if we expect it to be invalid.
-            auto closeTimeOffset = expectTxSetCloseTimeAffinity
-                                       ? (nextCloseTime - lclCloseTime)
-                                       : 0;
+            auto closeTimeOffset = nextCloseTime - lclCloseTime;
```

Comment:
wrong commit (this belongs to the next commit that removes support for not having ct affinity)